src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary/catalog.cljs

```diff
@@ -181,6 +179,12 @@
    :component-did-mount
    (fn [{:keys [props state refs]}]
      (let [{:keys [library-schema questions]} props]
+        (let [versions (get-in library-schema [:versions])]
+          (mapv (fn [version]
+                  (let [currentVersion (get-in library-schema [:properties (keyword version) :current])]
+                    (utils/log currentVersion)
+                    (swap! state update :attributes assoc (keyword version) currentVersion))) ;;js/parseInt should work?
```

Comment:
Version numbers are hidden when you're cataloging, getting saved behind the scenes, and then viewable in dataset attributes --> unless it's never been catalogued with a version # before. I also found a hack to get it to save properly as a number... our validation code is a little messed up